src/transmitter/types.ts

```diff
@@ -25,6 +25,7 @@ export interface IWorkloadMetadata {
 
 export interface IImageLocator extends IWorkloadLocator {
   imageId: string;
+  imageDigest: string;
```

Comment:
Is it possible to name it `imageWithDigest` just like it is on the `IScanImage`? I think just `imageDigest` conveys the message that it has the hash/digest but actually it also has the image name in here

I see that we have previously messed it up a little bit, adding fields like `imageName` which contain both the name _and the tag_...

Reply:
I tried to align it with previous view, but yeah seems totally legit to use `imageWithDigest`